tests/test_focused.py

```diff
@@ -0,0 +1,24 @@
+"""
+Test element "is_focused" functionality.
+"""
+
+from bok_choy.web_app_test import WebAppTest
+from .pages import FocusedPage
+
+
+class FocusedTest(WebAppTest):
+    """
+    Test query `is_focused` method and `focused` property.
+    """
+    def setUp(self):
+        super(FocusedTest, self).setUp()
+        self.page = FocusedPage(self.browser).visit()
+
+    def test_focused(self):
+        def el_focused():
+            return self.page.q(css="#main-content").focused
+
+        self.assertFalse(el_focused())
```

Comment:
Can you throw in a self.assertFalse(self.page.q(css="#i-dont-exist").focused) so that the test code also has an example showing the return value is False if the element does not exist?